cmake/CMakeDependencies.cmake

```diff
@@ -28,6 +29,7 @@ endif()
 
 if(${HTTP_CLIENT})
 # find CURL
+	hunter_add_package(CURL)
```

Comment:
When you're using Hunter it should be `find_package(CURL CONFIG REQUIRED)`:
* https://github.com/ruslo/hunter/wiki/pkg.curl